assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -37,6 +37,24 @@ che.workspace.auto_snapshot=true
 # Otherwise create a new workspace.
 che.workspace.auto_restore=true
 
+
+# Workspace threads pool configuration, this pool is used for workspace related
+# operations that require asynchronous execution e.g. starting/stopping/snapshotting
+
+# possible values are 'fixed', 'cached'
+che.workspace.pool.type=fixed
```

Comment:
@evoevodin has taken the right approach.

The way it works is that anything in che.properties can be overridden in the `che.env` with a naming convention.  So in this case, an admin can enter `CHE_WORKSPACE_POOL_TYPE=cached` and it will override the property.

However, we are only adding entries in the default `che.env` which would be entries that we commonly expect an administrator to need to edit.  For all other entries, which are more system operational internal entries, we just leave them in the che.properties and give admins a reference to it for investigation if they want.

I would include this property as an internal system property.